docs/modules/ROOT/pages/tutorials/getting-started-2.adoc

```diff
@@ -0,0 +1,72 @@
+= Getting Started with Project Syn (Part 2)
```

Comment:
It will not. It is not meant as a standalone document, but rather a followup of the first part.

Reply:
But then why split it up if it is not standalone? "Because the document gets too long" IMHO is a lame argument.

Reply:
It is **precisely** the argument. Length is a natural deterrent in any learning process. Besides, there will be a "Project Syn the Hard Way™®©" followup tutorial with much more in-depth. 

I refer to the original ticket that gave birth to this PR: 
As a CTO
I want to have a first impression in less than 30 minutes
So that I can evaluate Syn

Reply:
I also refer to research I'm reading about learning material design: https://twitter.com/akosma/status/1543530358670606340